src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
+    public String createReport(int supplySum, int buySum) {
+        StringBuilder reportBuilder = new StringBuilder();
+        return String.valueOf(reportBuilder.append("supply,").append(supplySum)
+                .append(System.lineSeparator())
+                .append("buy,").append(buySum).append(System.lineSeparator())
+                .append("result,").append(supplySum - buySum));
+    }
+
+    public void writeToFile(String toFileName, String data) {
+        try {
+            BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName, true));
+            bufferedWriter.write(data);
+            bufferedWriter.close();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't write data", e);
+        }
+    }
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplySum = 0;
+        int buySum = 0;
 
```

Comment:
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```
you program should look like this
methods should be independent 
readFromFile - just reads data from file
createReport - should create report, and return it as a string value
writeToFile - should only write to file